### PR TITLE
Zerolog replacement

### PR DESCRIPTION
**Pull request by pr2a**
#### Test Coverage Data

* Before
* After

### CODE REVIEW

**Comment by harmony-ek** [comment]
Jenkins speaking; can one of the admins verify this patch?

**Comment by harmony-ek** [comment]
Jenkins, test this please.

**Comment by LeoHChen** [comment]
@pr2a the PR failed the go fmt and goimport format, please fix it.

**Comment by LeoHChen** [comment]
Jenkins, test this please.

File: node/staking.go [hunk 1]
```diff
@@ -60,13 +60,19 @@ func (node *Node) UpdateStakingList(stakeInfoReturnValue *structs.StakeInfoRetur
 }
 
 func (node *Node) printStakingList() {
-	utils.GetLogInstance().Info("\n")
-	utils.GetLogInstance().Info("CURRENT STAKING INFO [START] ------------------------------------")
+	utils.Logger().Info().Msg("\n")
```
Comment (LeoHChen) [inline node/staking.go]: this is not needed.

File: node/staking.go [hunk 2]
```diff
@@ -60,13 +60,19 @@ func (node *Node) UpdateStakingList(stakeInfoReturnValue *structs.StakeInfoRetur
...
+			Str("BlsPubKey", hex.EncodeToString(stakeInfo.BlsPublicKey[:])).
+			Interface("BlockNum", stakeInfo.BlockNum).
+			Interface("lockPeriodCount", stakeInfo.LockPeriodCount).
+			Interface("amount", stakeInfo.Amount).
+			Msg("")
 	}
-	utils.GetLogInstance().Info("CURRENT STAKING INFO [END}   ------------------------------------")
-	utils.GetLogInstance().Info("\n")
+	utils.Logger().Info().Msg("CURRENT STAKING INFO [END}   ------------------------------------")
+	utils.Logger().Info().Msg("\n")
```
Comment (LeoHChen) [inline node/staking.go]: this is not needed.

**Comment by LeoHChen** [comment]
Jenkins, test this please.

**Pull request merged**